packages/flutter/lib/src/material/elevated_button.dart

```diff
@@ -45,6 +45,8 @@ import 'theme_data.dart';
 /// If [onPressed] and [onLongPress] callbacks are null, then the
 /// button will be disabled.
 ///
+/// Note: [ElevatedButton] is available after 1.22.0
```

Comment:
Avoid "note" (https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose)

Reply:
so what should I add here?

Reply:
@Hixie that would be cool if docs in general contained remarks of the first framework version some api was added, like android has in its docs

not in this way somewhere in the doc comment, but for example near the classname